### PR TITLE
[T] (quests) shop

### PR DESCRIPTION
**Pull request by luo-2233**
Co-authored-by: 疯子 <1161326997@qq.com>
Co-authored-by: 咸鱼啊喵丶 <74285624+XianYuAmiao@users.noreply.github.com>
Co-authored-by: Cactusstudent <97865546+Cactusstudent@users.noreply.github.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/shop.snbt [hunk 1]
```diff
@@ -36,11 +36,11 @@
 			}]
 		}
 		{
-			title: "Raw Materials Lootbox"
+			title: "原材料战利品箱"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "粗金属战利品箱"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 2]
```diff
@@ -56,7 +56,7 @@
 			}]
 		}
 		{
-			title: "Bees Items Lootbox"
+			title: "蜜蜂物品战利品箱"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]: 这翻译咋看咋奇怪，但是还不知道咋改

File: config/ftbquests/quests/chapters/shop.snbt [hunk 3]
```diff
@@ -169,8 +169,8 @@
 			}
 			x: 3.5d
 			y: 5.0d
-			subtitle: "Diamond 3"
-			description: ["Used to fast travel to your spawn! Mine the air for what feels like ages :)"]
+			subtitle: "3钻石币"
+			description: ["用于快速前往您的重生点！ 我在空气中挖掘岁月的痕迹 :)"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			description: ["用于快速回到您的重生点！我在空气中挖掘岁月的痕迹 :)"]
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 4]
```diff
@@ -371,11 +371,11 @@
 			}]
 		}
 		{
-			title: "Mob Drops Lootbox"
+			title: "生物资源战利品盒"
 			icon: "minecraft:rotten_flesh"
 			x: 14.5d
 			y: -5.5d
-			subtitle: "1 Iron"
+			subtitle: "1 铁币"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			subtitle: "1铁币"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 5]
```diff
@@ -390,11 +390,11 @@
 			}]
 		}
 		{
-			title: "Artifacts"
+			title: "遗物"
 			icon: "artifacts:crystal_heart"
 			x: 7.5d
 			y: -5.5d
-			subtitle: "2 Netherite"
+			subtitle: "2 下界合金币"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			subtitle: "2下界合金币"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 6]
```diff
@@ -390,11 +390,11 @@
 			}]
 		}
 		{
-			title: "Artifacts"
+			title: "遗物"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "饰品"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 7]
```diff
@@ -429,11 +429,11 @@
 			}]
 		}
 		{
-			title: "Oak logs"
+			title: "橡木原木"
 			icon: "minecraft:oak_log"
 			x: 3.5d
 			y: -13.0d
-			subtitle: "5 Copper"
+			subtitle: "5 铜币"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			subtitle: "5铜币"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 8]
```diff
@@ -449,11 +449,11 @@
 			}]
 		}
 		{
-			title: "Dirt"
+			title: "泥土"
 			icon: "minecraft:dirt"
 			x: 3.5d
 			y: -11.5d
-			subtitle: "1 Gold"
+			subtitle: "1 金币"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			subtitle: "1金币"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 9]
```diff
@@ -478,7 +478,7 @@
 			}
 			x: 3.5d
 			y: -1.0d
-			subtitle: "5 Copper"
+			subtitle: "5 铜币"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			subtitle: "5铜币"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 10]
```diff
@@ -494,11 +494,11 @@
 			}]
 		}
 		{
-			title: "Steak x64"
+			title: "牛排 x64"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "牛排x64"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 11]
```diff
@@ -514,11 +514,11 @@
 			}]
 		}
 		{
-			title: "Steak x8"
+			title: "牛排 x8"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "牛排x8"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 12]
```diff
@@ -2076,11 +2076,11 @@
 			}]
 		}
 		{
-			title: "Unlucky_Blocks x16"
+			title: "幸运方块 x16"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "幸运方块x16"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 13]
```diff
@@ -974,11 +974,11 @@
 			}]
 		}
 		{
-			title: "Gems Lootbox"
+			title: "Gems战利品箱"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "宝石战利品箱"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 14]
```diff
@@ -914,11 +914,11 @@
 			}]
 		}
 		{
-			title: "Pretty Plants Lootbox"
+			title: "美丽植物战利品箱"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "漂亮的植物战利品箱"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 15]
```diff
@@ -875,11 +875,11 @@
 			}]
 		}
 		{
-			title: "Iron Ingot x16"
+			title: "铁锭 x16"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "铁锭x16"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 16]
```diff
@@ -855,11 +855,11 @@
 			}]
 		}
 		{
-			title: "Diamond x16"
+			title: "钻石 x16"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "钻石x16"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 17]
```diff
@@ -704,11 +704,11 @@
 			}]
 		}
 		{
-			title: "Pancake x32"
+			title: "煎饼 x32"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "煎饼x32"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 18]
```diff
@@ -684,11 +684,11 @@
 			}]
 		}
 		{
-			title: "Redstone Block x16"
+			title: "红石块 x16"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "红石块x16"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 19]
```diff
@@ -658,7 +658,7 @@
 			}]
 		}
 		{
-			title: "Full Chocolate Bar x1"
+			title: "巧克力板 x1"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "巧克力板x1"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 20]
```diff
@@ -639,11 +639,11 @@
 			}]
 		}
 		{
-			title: "Scathed Disc 1"
+			title: "破损的唱片 1"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "破损的唱片x1"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 21]
```diff
@@ -574,11 +574,11 @@
 			}]
 		}
 		{
-			title: "Swords Lootbox"
+			title: "武器战利品盒"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "武器战利品箱"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 22]
```diff
@@ -594,11 +594,11 @@
 			}]
 		}
 		{
-			title: "Furniture Lootbox"
+			title: "家具战利品盒"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "家具战利品箱"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 23]
```diff
@@ -554,11 +554,11 @@
 			}]
 		}
 		{
-			title: "Dye Lootbox"
+			title: "染料战利品盒"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "染料战利品箱"
```

File: config/ftbquests/quests/chapters/shop.snbt [hunk 24]
```diff
@@ -371,11 +371,11 @@
 			}]
 		}
 		{
-			title: "Mob Drops Lootbox"
+			title: "生物资源战利品盒"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "生物资源战利品箱"
```

**Review by Cactusstudent — CHANGES_REQUESTED** [review]

File: config/ftbquests/quests/chapters/shop.snbt (hunk 2 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/shop.snbt]:
```suggestion
			title: "养蜂战利品箱"
```

**Review by AnzhiZhang — APPROVED** [review]

**Review by Cactusstudent — APPROVED** [review]

**Pull request merged**